.pre-commit-config.yaml

```diff
@@ -33,7 +33,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
5.10.1 had a bug in it that wouldn't allow it to install properly in the pre-commit environment.